src/main/scala/com/gu/acquisition/services/DefaultAcquisitionService.scala

```diff
@@ -0,0 +1,35 @@
+package com.gu.acquisition.services
+
+import cats.data.EitherT
+import com.gu.acquisition.model.AcquisitionSubmission
+import com.gu.acquisition.model.errors.AnalyticsServiceError
+import com.gu.acquisition.typeclasses.AcquisitionSubmissionBuilder
+import okhttp3.OkHttpClient
+
+import scala.concurrent.ExecutionContext
+
+class DefaultAcquisitionService(implicit client: OkHttpClient) extends AcquisitionService {
+  private val ophanService = new OphanService()
+  private val gAService = new GAService()
+  override def submit[A: AcquisitionSubmissionBuilder](a: A)(implicit ec: ExecutionContext) = {
+    val ov = ophanService.submit(a).value
+    val gv = gAService.submit(a).value
+    val result = for {
+      o <- ov
```

Comment:
nitpicking but do we need to define these vals outside the `for` comprehension?

Reply:
I believe we need to create them outside the for comprehension in order for them to run in parallel https://alvinalexander.com/scala/how-use-multiple-scala-futures-in-for-comprehension-loop

Reply:
Makes sense